lib/circuitbox/circuit_breaker.rb

```diff
@@ -186,6 +188,11 @@ def log_metrics(error_rate, failures, successes)
       n.metric_gauge(:success_count, successes)
     end
 
+    def log_circuit_execution_time(time)
+      n = notifier.new(service)
```

Comment:
I have a PR open to change this so we won't need to create a new notifier every time. So depending on when that gets merged this line might need some changes, just a heads up.